### PR TITLE
Civ 1281 contact us for help

### PR DESCRIPTION
**Pull request by laarmada**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-1281

### Change description ###

Each webpage must include a 'Contact us for help' item. When this item is selected, it will expand to display the assistance information. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[x ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/views/features/response/citizenDob/citizen-dob.njk [hunk 1]
```diff
@@ -77,10 +77,14 @@
              }) }}
           </div>
             {{ govukButton({ text: t("Save and continue") }) }}
-            {{ contactUsForHelp(t) }}
           </form>
         </div>
       </div>
+      <div class="govuk-grid-row">
```
Comment (TatsyMaslova) [inline src/main/views/features/response/citizenDob/citizen-dob.njk:83]: would this also apply to phone number view?

Comment (laarmada) [reply src/main/views/features/response/citizenDob/citizen-dob.njk:83]: yes sure. Fix it!

File: src/main/views/macro/contact-us-for-help.njk [hunk 2]
```diff
@@ -1,20 +1,41 @@
...
+            <h3 class="govuk-heading-m">{{name}}</h3>
+            {% if phoneNumber!=='' %}
+                   {{ t(phoneNumber) }}
+            {% endif %}
+            {% if spanContent!=='' %}
+                <p><span class="block">{{ t(spanContent) }}</span></p>
+            {% endif %}
+            {% if href!=='' %}
+                <a href={{href}} class="govuk-details__summary">{% if hrefContact!=='' %}{{hrefContact}}</a><br/>{% endif %}{{t(hrefContent)}}</a></p>
+            {% endif %}
```
[the remark this answers is not in the document]
Comment (laarmada) [reply src/main/views/macro/contact-us-for-help.njk:23]: ok cool, thanks Taoufik

**Review by laarmada — COMMENTED** [review]
all comments solved

**Review by LeonardoPalmeiro — CHANGES_REQUESTED** [review]

File: src/main/views/macro/contact-us-for-help.njk [hunk 3]
```diff
@@ -1,20 +1,41 @@
...
+{% set webChatUnavailable = t('The web chat service is temporarily unavailable, please try again later.') %}
+{% set callCharges = t('Find out about call charges (opens in a new window)') %}
+{% set openingHours = t('Monday to Friday, 8.30am to 5pm.') %}
+
+{% macro _widget(spanContent, name, href, hrefContact, hrefContent, phoneNumber) %}
+            <h3 class="govuk-heading-m">{{name}}</h3>
+            {% if phoneNumber!=='' %}
+                   {{ t(phoneNumber) }}
+            {% endif %}
+            {% if spanContent!=='' %}
```
[the remark this answers is not in the document]
Comment (laarmada) [reply src/main/views/macro/contact-us-for-help.njk:18]: let me check..

File: src/main/views/features/response/ageEligibility/under-18.njk [hunk 4]
```diff
@@ -9,24 +9,11 @@
...
-          <p class="govuk-body">0300 123 7050<br />
-            {{ t('Monday to Friday') }}, 8.30am to 5pm.<br />
-            <a href='https://www.gov.uk/call-charges' target='_blank'>{{ t('Find out about call charges (opens in a new window)') }}</a>
-          </p>
-          <p class="govuk-body">
-            {{t('You might be able to get advice about the claim from ')}}
-            <a href="https://www.citizensadvice.org.uk/about-us/contact-us/" target='_blank'>{{ t('Citizens Advice (opens in a new window)') }}</a>
-          </p>
+       <div class="govuk-grid-row">
+           <div class="govuk-grid-column-three-thirds">
```
Comment (TatsyMaslova) [inline src/main/views/features/response/ageEligibility/under-18.njk:13]: this does not  have contact  us for help. i think you need to revert the change. it has different requirements

Comment (TatsyMaslova) [reply src/main/views/features/response/ageEligibility/under-18.njk:13]: think it will break what Mounika already tested https://tools.hmcts.net/jira/browse/CIV-1107.

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by TatsyMaslova — APPROVED** [review]
